pkgs/by-name/sp/speechd/package.nix

```diff
@@ -77,9 +77,11 @@ stdenv.mkDerivation rec {
       libsndfile
       libao
       libpulseaudio
-      alsa-lib
       python
     ]
```

Comment:
Looks like it also needs `libsystemd` (`systemdMinimal` package should probably work).